competency-explorer/src/components/FilterBar/LevelFilters/LevelFilter/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback } from 'react';
+import cx from 'classnames';
+import { Level } from '../../../../models';
+import { useSelector, useDispatch } from 'react-redux';
+import { createFilterAction } from '../../../../reducers';
+import './styles.scss';
+import { selectLevelVisibility } from '../../../../selectors/visibility';
+
+export type LevelFilterProps = {
+    level: Level
+}
+
+export const LevelFilter = ({ level }: LevelFilterProps) => {
+    const isVisible = useSelector((s) => selectLevelVisibility(s, level))
+    const dispatch = useDispatch();
+    
+    const onClick = useCallback(() => {
+        dispatch(createFilterAction(level, 'TOGGLE_LEVEL'))
+    }, [isVisible])
+
+    return (
+        <div className={cx('levelFilter', !isVisible && 'hidden')} onClick={onClick}>{level}</div>
```

Comment:
I think it would have been more maintainable from our side to use [CSS Modules per CRA's support](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/), since our other apps use it and we've started the process of looking at CSS-in-JS seriously. But I can understand if you don't have the time to port over 😄 , not a blocker.

Reply:
a future PR for future Kip (or future anyone)